usagov/vote-gov-nvrf-app
# VOTE-2784 Add a new vite entry point for Spanish

Ticket: https://cm-jira.usa.gov/browse/VOTE-2784

Added an entry point at /es/ to test Spanish locally without having to modify index.html

Test steps:
1. Start the app locally
2. Navigate to http://localhost:5173/es/ (the ending backslash is important)
3. Confirm the Spanish content correctly loads into the app for all steps and fields
